Refine group management docs (#520)
pathoplexus/pathoplexus: monorepo/website/src/pages/docs/how-to/create-group.mdx

If your organization does not yet have a group on Pathoplexus, you can create one. 

If they already have a group, please do not create another! Contact someone in that group and ask them to add you using your Pathoplexus username.

Please remember that everyone within a group is able to submit, edit, and revoke sequences uploaded by anyone else in that group. Read more about that in [Adding or removing a user to or from a group](/docs/how-to/add-remove-user-group).

1. Log into your account and enter your account page by clicking 'My account' in the top-right corner of the webpage
2. Click 'Create Group'
3. Fill in the required information about your group
4. Click 'Submit' to create your group

> You can now add members to your group by following the steps in [the docs on adding or removing a user to or from a group](/docs/how-to/add-remove-user-group)

Author: ```suggestion
You can now add members to your group by following the steps [outlined in the docs](/docs/how-to/add-remove-user-group)
```
